runtime/ftplugin/lua.lua

```diff
@@ -1,4 +1,8 @@
 -- use treesitter over syntax
 vim.treesitter.start()
 
-vim.b.undo_ftplugin = (vim.b.undo_ftplugin or '') .. '\n call v:lua.vim.treesitter.stop()'
+vim.bo.includeexpr = "v:lua.require'_ftplugin.lua'.includeexpr(v:fname)"
```

Comment:
Is `_ftplugin.lua` needed because `runtime/ftplugin/` is not on 'runtimepath' ? 

Rather than introducing `_ftplugin.lua`, can we use `loadfile(vim.env.VIMRUNTIME..'/ftplugin/lua.lua')` ?  This should work:

    vim.bo.includeexpr = "v:lua.loadfile($VIMRUNTIME..'/ftplugin/lua.lua')"

Reply:
No, it's just an arbitrary path to avoid polluting the global namespace (and the top-level `runtime/lua` directory). It's not about _this_ `ftplugin` but about the "autoload" file for the includeexpr (which needs to live in a different file for this to work without headaches).

You can replace `require` with `loadfile`, but this will give you no benefit (and cost you `require`'s caching).